salt/edx/prod.sls

```diff
@@ -34,11 +34,10 @@ configure_python_ppa_for_edx:
 install_os_packages:
   pkg.installed:
     - pkgs:
-        - git
```

Comment:
Correct, the 1: prefix isn't required.

On Aug 9, 2016 5:44 PM, Brandon DeRosier <notifications@github.com> wrote:

In salt/edx/prod.sls<https://github.com/mitodl/salt-ops/pull/150#discussion_r74150589>:

> @@ -34,11 +34,10 @@ configure_python_ppa_for_edx:
>  install_os_packages:
>    pkg.installed:
>      - pkgs:
> -        - git


I found the package in the ppa here: https://launchpad.net/~git-core/+archive/ubuntu/ppa<https://launchpad.net/%7Egit-core/+archive/ubuntu/ppa>

the 1: piece is not necessary, right? Maybe silly, but I just don't know what it means. :)

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/mitodl/salt-ops/pull/150/files/e31548d87ed101d0dc058ca036061c268a386bd8#r74150589>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AAdPcP2JbZGWwuUqC2eIlXdmh_gYxQblks5qePTGgaJpZM4JgeSL>.